### DISCUSSION TITLE
Proposal for an receiver with an integrated model finder

### DISCUSSION THREAD

**Discussion by FranzPVal** [ExpressLRS/ExpressLRS#1345] [Ideas]
Subject:
battery supported long-range receiver equipped with an buzzer and flashing bright LED having unique capabilities for finding a lost model

Function:
The receiver stores valid/good GPS position data given by the telemetry continuously and as a separate set in the moment the model is first armed (AUX1 0→1; home position). After a 5V power lost the receiver turns itself off, if the current position is near enough to the home position (constant parameter – can be altered before flashing). Otherwise the receiver stay operational using its own battery until a (shutdown) button is held for 3s and released and pressed short again (preventing mechanical or electrical shorts in a crash). While the receiver is in this backup state if will reconnect to the transmitter and sending telemetry data (GPS position) back. With an assigned AUX (constant parameter) the internal buzzer + flash LED can be activated.

Scenario:
If there is a problem with the model in a long-range flight at it goes down far away from home position its usual that the radio and video link is lost before the exact position of the model was send back. In this case the owner starts searching on the last known position. If the main battery was disconnected or depleted after the crash there is a chance never finding the model even with an model-finder buzzer. For this case the working receiver kicks in: If the owner gets close enough the radio link reestablish and the shown GPS position updates. Now he can start searching on the spot using RSSI and the buzzer (by flipping the switch).

Hardware:
A ELRS receiver working this way must contain additional components: A 3.3V low-dropout regulator with an enable input, a 5.0V LiPo charging IC and 1-2 switching MosFets driving a integrated buzzer and a bright LED. The RX µC can control the regulator enable input (keeping it on w/o 5V) using a GPIO. The regulator is also held active by the presence of the 5V external supply. A ADC input from the RX µC can be used to watch the battery voltage. A button (for shutdown) is connected to a GPIO too.

**Comment by Yury-MonZon** [comment]
How about just the software part to send last gps coordinates once a minute or so?

**Comment by malarcky** [comment]
Is anything on this planned? Really like this feature.
